### PR TITLE
Adding first draft of FAQ vignette

### LINKED ISSUE #77: Write FAQ Vignette
In progress!

### PR DESCRIPTION
**Pull request by skirmer**
Added a vignette (sort of a stub but it can stand alone for now) for FAQ purposes. It's built, so if all goes right it should be fully integrated with the package. Will resolve  #77 .

### CODE REVIEW

File: vignettes/FAQ.Rmd [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## Query Returns No Results
+
+After you have verified with certainty that your query is appropriately structured and written, you might still have challenges. What to do if you get the error `Query is syntactically valid but 0 documents were matched. Returning NULL` ?
+
+### Troubleshooting Guide
+
+* Are your **search terms** named and described correctly? All spelled right?  
+* Are you looking in the **correct index**? Perhaps your document is in a different one.  
+* If you are passing **dates or datetimes**, are the formats of your input formatted just right? 
+* IMPORTANT: **Are the terms you are using indexed**? This is a tricky one. Your term may exist and have data in the documents, but if your particular elasticsearch database has not indexed that term, you won't be able to use it for searching. This does not mean that the document/s aren't there, but just that you can't use that term for searching.
```
Comment (austin3dickey) [inline vignettes/FAQ.Rmd]: I think the "you must use only lowercase when using `terms` to search analyzed fields" trips people up a lot. Could you call that out?

**Comment by jameslamb** [comment]
@skirmer This looks good to me! Once you add the change @austin3dickey asked for, we can merge it up.

Thanks again for doing this

**Comment by jameslamb** [comment]
@skirmer we're ready to merge this whenever you have a chance to address @austin3dickey 's request!

**Comment by jameslamb** [comment]
@austin3dickey @skirmer 

I am going to just merge this. Since we were only asking for a small addition, we can always add that addition later.

**Pull request merged**